app/styles/vars.css

```diff
@@ -90,6 +90,7 @@ html.dark {
   --bg-mode-active: var(--theme-transparent);
   --abilities-button-bg: hsl(237.3deg 42.3% 26.6%);
   --border: hsl(237.3deg 42.3% 45.6%);
+  --border-radius-max: 9999px;
```

Comment:
I don't think CSS var is needed for this

Reply:
Fair enough, I was just porting CSS over from the Splatoon 2 user search page & this was one of the properties that was specified. I don't think it's needed either & will remove it

Reply:
Okay, turns out that any kind of border-radius makes it round (so long as it is specified), so I went ahead with that in the `u.css` file instead